app/src/main/java/com/nutomic/syncthingandroid/fragments/FolderListFragment.java

```diff
@@ -101,9 +102,15 @@ private void updateList() {
 
     @Override
     public void onItemClick(AdapterView<?> adapterView, View view, int i, long l) {
+        Folder folder = mAdapter.getItem(i);
+        FolderStatus folderStatus = mAdapter.mLocalFolderStatuses.get(folder.id);
+        boolean overridableChanges = folder.type.equals(Constants.FOLDER_TYPE_SEND_ONLY)
+            && folderStatus.isNeedsItems()
+            && folderStatus.isOutOfSync();
```

Comment:
The previous condition checked for folder idleness. That's because you can't revert if a scan or sync operation is in progress. This is a bit of a problem with the menu, as the time when you open the menu is not necessarily the time the you click override. So the enabledness of the button might not match reality. Which means the operation will not work (likely hang and time out eventually).

Reply:
Except I just saw the condition for `isOutOfSync` further down - it checks both `isNeedsItems` and for idleness. So the `isNeedsItems` check here is redundant. 

The timing problem in the menu still is a thing. Ideally that would somehow be hooked to folder status, but that's probably not super simple? I may be able to be convinced to just close my eyes and pretend I didn't know about this.

Reply:
I removed the isNeedsItems as redundant.